### PR TITLE
runbot migrate to new api

### PR DESCRIPTION
**Pull request by rga-odoo**
task: https://www.odoo.com/web#id=24267&view_type=form&model=project.task&action=327&menu_id=4720

### CODE REVIEW

File: runbot/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,363 @@
+import itertools
+import hashlib
+from collections import OrderedDict
+
+import operator
+import werkzeug
+from matplotlib.font_manager import FontProperties
+from matplotlib.textpath import TextToPath
```
Comment (rim-odoo) [inline runbot/controllers/main.py]: seems overkill

**Comment by KangOl** [comment]
Can you rebase your work with latest commits?

**Comment by rga-odoo** [comment]
@KangOl  I have rebased the branch with the latest commit

**Comment by mart-e** [comment]
runbot merged to v11 and moved to https://github.com/odoo/runbot/ :tada:

**Pull request closed without merging**